enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -243,8 +243,9 @@
                         :dataset_query       (s/eq {})
                         :parameter_mappings  (s/eq [{:parameter_id "deadbeef"
                                                      :card_id      c1-eid
-                                                     :target [:dimension [:field ["My Database" nil "Schemaless Table" "Some Field"]
-                                                                          {:source-field ["My Database" "PUBLIC" "Schema'd Table" "Other Field"]}]]}])
+                                                     :source_type  "field"
```

Comment:
Parameters (everywhere) now have a `source_type` key that gets defaulted to `"field"` (the current behavior). So there's some test noise to put that in to all the expectations.